### PR TITLE
Switch to service account

### PR DESCRIPTION
**Pull request by penalosa**
Switch the pre-release registry away from @GregBrimble's personal account to use a proper service account (cc @workers-devprod)

### CODE REVIEW

File: .github/workflows/pullrequests.yml
Comment (GregBrimble) [inline .github/workflows/pullrequests.yml:64]: `env_var_v2_final_(1).docx`

Any way we can just re-use the old name? Fine to break the prerelease registry for a moment while getting this out if needed.

**Review by GregBrimble — APPROVED** [review]

File: .github/workflows/pullrequests.yml
Comment (penalosa) [reply .github/workflows/pullrequests.yml:64]: Absolutely—just didn't want to break things. I suppose it's okay to break the pre-release registry for a bit, so I can rename them just before merging

**Comment by penalosa** [comment]
On hold pending internal domain changes

**Review by rozenmd — APPROVED** [review]

**Pull request merged**
